cmd/workspace.go

```diff
@@ -198,6 +203,85 @@ func newWorkspaceUserListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+/*
+	Workspace teams commands
+*/
+
+func newWorkspaceTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Workspace Team resources",
+		Long:  "Teams can be added or removed from Workspaces",
+	}
+	cmd.AddCommand(
+		newWorkspaceTeamAddCmd(out),
+		newWorkspaceTeamUpdateCmd(out),
+		newWorkspaceTeamRmCmd(out),
+		newWorkspaceTeamListCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add TEAM",
+		Short:   "Add a Team to a Workspace",
+		Long:    "Add a Team to a Workspace",
+		Example: "astro workspace team add <team-id> --workspace-id <workspace-id> --role <role>",
```

Comment:
In the CLI Convergence, we want to standardize this CLI project and all commands.

We have decided to avoid using arguments and use `flags` instead, can you change the team-id arg to a `--team-id` flag, and add validation please ?

Reply:
True, I updated the [standardization document](https://www.notion.so/astronomerio/CLI-Convergence-Plan-566d2a249c3c4cf9b75e792514ad8023#4d68e54c1514460a9d3050a91ada6739) to reflect these.

In this standardization document, we decided we want to only use flags for create/add commands like in this example.

Can you move the <team-id> argument to a flag `--team-id` please ?

Reply:
Sure, thanks for the clarification. Shall reflect these accordingly

Reply:
@AmFlint Also will this flag apply to all CRUD commands or do we want to keep this for the create command only?